Model/ImportExport/Processor/Import.php

```diff
@@ -71,8 +90,27 @@ private function validateData(array $data): void
             $this->nodeProcessor->validateImportData($data[ExtendedFields::NODES]);
         }
 
-        if ($this->validationAggregateError->getErrors()) {
+        if (empty($this->scopeConfig->getValue('snowmenu/import/strip_invalid_nodes')
```

Comment:
By default == 1, can be changed by adding it to app/etc/env.php or we can add system.xml entry